app/models/cohort.rb

```diff
@@ -1,51 +1,59 @@
 # frozen_string_literal: true
 
 class Cohort < ApplicationRecord
-  has_many :schedules, class_name: "Finance::Schedule"
-  has_many :partnerships
-  has_many :statements
   has_many :call_off_contracts
   has_many :npq_contracts
+  has_many :partnerships
+  has_many :schedules, class_name: "Finance::Schedule"
+  has_many :statements
+
+  # Class Methods
+
+  def self.active_registration_cohort
+    where(registration_start_date: ..Date.current).order(start_year: :desc).first
+  end
 
   def self.current
-    # TODO: Register and Partner 262: Figure out how to update current year
-    if FeatureFlag.active?(:multiple_cohorts)
-      where("academic_year_start_date <= ?", Time.zone.now).order(start_year: :desc).first
-    else
-      find_by(start_year: 2021)
-    end
+    date_range = (Date.current - 1.year + 1.day)..Date.current
+    where(academic_year_start_date: date_range).first
   end
 
   def self.next
-    year = FeatureFlag.active?(:multiple_cohorts) ? 2022 : 2021
-    find_by(start_year: year)
+    date_range = (Date.current + 1.day)..(Date.current + 1.year)
+    where(academic_year_start_date: date_range).first
   end
 
-  def self.active_registration_cohort
-    where("registration_start_date <= ?", Time.zone.now).order(start_year: :desc).first
+  def self.previous
+    date_range = (Date.current - 2.years + 1.day)..(Date.current - 1.year)
+    where(academic_year_start_date: date_range).order(start_year: :desc).first
   end
 
-  def description
-    "#{start_year} to #{start_year + 1}"
-  end
+  # Instance Methods
 
-  def next
-    Cohort.find_by(start_year: start_year + 1)
+  # e.g. "2021/22"
+  def academic_year
+    sprintf("#{start_year}/%02d", ((start_year + 1) % 100))
```

Comment:
Might need to be careful here about displaying a date range with a `/`, the GDS guidelines suggest using `2022 to 2023`.

However, depending on where we display it (ie if there are several in a table) this might be ok.

https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#dates

Reply:
Actually as we're not changing the behaviour here ignore this, I'll revisit it separately.